README.md

```diff
@@ -34,19 +65,72 @@ You can run the local UI through [joshsim-server](https://language.joshsim.org/d
 $ java -jar joshsim-server.jar
 ```
 
-This will start a local web server which makes the UI available via your browser where you can work in private.
+This will start a local web server which makes the UI available via your browser where you can work in private. The local server optionally supports a sandbox mode that limits access to only the code and jshd files provided, with no network access otherwise.
 
 ### Containerized usage
 Containerization through [Docker](https://www.docker.com) and [Development Containers](https://containers.dev) can help you move your work from one computer to the next with ease. Please see our `Dockerfile` and `devcontainer.json`.
 
 ### Distributed usage
-Distributing workloads is easy. Simply deploy either our jar file or container to a serverless solution like [Labmda](https://aws.amazon.com/lambda/) / [CloudRun](https://cloud.google.com/run) or submit on your own cluster via [Kubernetes](https://kubernetes.io). You can send the Josh jar over the network to get your script and you can write to cloud storage. All you have to do is provide the command line arguments:
+Distributing workloads is easy. Simply deploy either our jar file or container to a serverless solution like [Lambda](https://aws.amazon.com/lambda/) / [CloudRun](https://cloud.google.com/run) or submit on your own cluster via [Kubernetes](https://kubernetes.io). The Josh server supports HTTP2 for efficient communication. You can send the Josh jar over the network to get your script and you can write to cloud storage.
+
+For results returned via HTTP2 streaming back to the user client:
 
 ```
-$ java -jar joshsim.jar run https://your-url.org/script.josh --http-basic-user USERNAME --http=-basic-pass PASSWORD --simulation TestSimulation --replicates 10 --output minio://your-s3-bucket/test_simulation.avro --minio-key ACCESS_KEY --minio-secret ACCESS_SECRET
+$ java -jar joshsim.jar run https://your-url.org/script.josh --http-basic-user USERNAME --http-basic-pass PASSWORD --simulation TestSimulation --replicates 10
+```
+
+For results saved to an output location (note that avro dumping is currently limited):
```

Comment:
Let’s remove the output location example for now. We will document it in a later PR.